packages/react/src/components/form/Select/Select.tsx

```diff
@@ -19,20 +19,12 @@ export type SelectProps = MergeRight<
 >;
 
 export const Select = forwardRef<HTMLSelectElement, SelectProps>(
-  function Select({ className, onKeyDown, onMouseDown, ...rest }, ref) {
+  function Select({ className, disabled, ...rest }, ref) {
     return (
       <select
         className={cl('ds-input', className)}
+        disabled={disabled || rest.readOnly}
         ref={ref}
-        onKeyDown={(event) => {
-          if (event.key === 'Tab') return;
-          if (rest.readOnly) event.preventDefault(); // Make readonly work for select
-          onKeyDown?.(event);
-        }}
-        onMouseDown={(event) => {
-          if (rest.readOnly) event.preventDefault(); // Make readonly work for select
-          onMouseDown?.(event);
-        }}
         {...rest}
       />
     );
```

Comment:
Doing this makes us undo https://github.com/digdir/designsystemet/issues/1659, which we decided on earlier this year to implement.

Reply:
Agreed. Should we include a designer as this affect user experience?